sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -551,13 +560,18 @@ public Secret restoreSecret(byte[] backup) {
      * call {@link SecretClient#getSecret(SecretBase)} . This will return the {@link Secret} secrets with values included of the specified versions.</p>
      * {@codesnippet com.azure.security.keyvault.secretclient.listSecretVersions#string-Context}
      *
+     *
+     * <p>It is possible to get full Secrets with values for each version from this information. Loop over all the {@link SecretBase secret} by each page and
```

Comment:
Description is fine, But the header should stay too.
`<p><strong>Code Samples to iterate over secret versions by page</strong></p>`